docs/typescript.md

```diff
@@ -0,0 +1,53 @@
+# Typescript
+
+This guide is an example of how to develop a function in Typescript with
+the Functions Framework.
+
+1. Use [gts](https://github.com/google/gts) to configure Typescript.
+
+    ```sh
+    npx gts init
+    ```
+
+2. Install the required packages:
+
+    ```sh
+    npm install @google-cloud/functions-framework
+    # an HTTP signature requires express types
+    npm install @types/express --save-dev
+    # https://www.npmjs.com/package/tsc-watch
+    npm install tsc-watch --save-dev
+    ```
+
+3. Add a `start` script to `package.json`, passing in the `--source` flag to
+   point to the compiled code directory (configured by `gts` in this example).
+   Also add a `watch` script to use for development:
+
+    ```js
+      "scripts": {
+        "start": "functions-framework --source=build/src/ --target=helloWorld",
+        "watch": "tsc-watch --onSuccess 'npm start'",
+        ...
+      }
+    ```
+
+4. Create a `src/index.ts` file with the following contents:
```

Comment:
Looks like `gts` already creates this file when I tried it.

Say "Replace" instead of "Create".